src/Compilers/VisualBasic/Test/Emit/PDB/PDBTests.vb

```diff
@@ -6697,6 +6694,45 @@ End Module
 </symbols>)
         End Sub
 
+        <WorkItem(841361, "DevDiv")>
+        <Fact()>
+        Public Sub SequencePointsInAQuery_32()
+            Dim source =
+<compilation>
+    <file><![CDATA[
+Imports System.Collections
+Imports System.Collections.Generic
+Imports System.Linq
+
+Module Module1
+    Sub Main()
+        Dim x = From a in {1, 2, 3}
+                Let b = a * a
```

Comment:
The current test cases like SequencePointsInAQuery_04 already cover the multiple cases. With this bug, their PDBs have also been updated.